packages/expo-video/ios/VideoView.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023-present 650 Industries. All rights reserved.
+
+import AVKit
+import ExpoModulesCore
+
+public final class VideoView: ExpoView, AVPlayerViewControllerDelegate {
+  lazy var playerViewController = AVPlayerViewController()
+
+  var player: AVPlayer? {
+    didSet {
+      playerViewController.player = player
+    }
+  }
+
+  var isFullscreen: Bool = false
+
+  public override var bounds: CGRect {
+    didSet {
+      playerViewController.view.frame = self.bounds
+      print("bounds didSet:", self.bounds)
+    }
+  }
+
+  public required init(appContext: AppContext? = nil) {
+    super.init(appContext: appContext)
+
+    clipsToBounds = true
+    playerViewController.delegate = self
+    playerViewController.view.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+    playerViewController.view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
```suggestion
```
Don't think we need to disable the autoresizing mask here. We're not adding any other constraints manually.